compiler/luci-interpreter/src/kernels/Add.test.cpp

```diff
@@ -27,14 +28,22 @@ namespace
 
 using namespace testing;
 
+class AddTest : public ::testing::Test
+{
+protected:
+  void SetUp() override { _memory_manager = std::make_unique<TestMemoryManager>(); }
```

Comment:
Note that there are no need to move or hide memory manager from someone in this code, so you can simply use TestMemoryManager as a member instead of unique_ptr to it.

This will save you some lines of code =)

Reply:
P.s. Do not rewrite this code, just a tip for the future